Samples/README.md

```diff
@@ -0,0 +1,9 @@
+# Decoder Samples
+
+The following samples are available:
+
+- Decoder
+- UniversalDecoder
+- CayenneDecoder
+
+The documentation can be found at <https://github.com/Azure/iotedge-lorawan-starterkit/tree/docs/main/docs/samples>
```

Comment:
Why not send to the documentation URL instead of the repo?

```suggestion
The documentation can be found at <https://azure.github.io/iotedge-lorawan-starterkit/user-guide/documentation/>
```

Reply:
fixed by adding the links to the list items themselves